.travis.yml

```diff
@@ -65,3 +61,9 @@ script:
 
 after_success:
 - bash <(curl -s https://codecov.io/bash)
+
```

Comment:
👍 I thought about asking your thoughts on stages last week.  
What are your thoughts on moving the tests to a stage?  
Example:

```yaml
- stage: test
  name: iPhone X 11.4 tests
  script: 
  - (destination test...)
- stage:
  name: iPhone X 12.1 tests
  script: 
  - (destination test...)
- stage:
  name: Apple TV 12.2 tests
  script: 
  - (destination test...)
```